src/sail_json_backend/json.ml

```diff
@@ -202,6 +204,24 @@ let add_assembly app_id p =
       Hashtbl.add assembly (string_of_id app_id) x
     end
 
+
+
+let json_of_registers () =
+  let regs = Hashtbl.fold (fun name regtype accum ->
+    (name, regtype) :: accum
+  ) registers [] in
+  let sorted_regs = List.sort (fun (_, t1) (_, t2) -> String.compare t1 t2) regs in
```

Comment:
This sorts by regtype only. Could you add a secondary sort, by regname? I think without this, you may get registers in odd orders, no? Like "x0", "x4", "x17", "x1", ...?